CHANGELOG.md

```diff
@@ -3,8 +3,47 @@
 <!-- prettier-ignore-start -->
 
 
-## 0.7.13
+## 0.7.14
+
+### Enhancements
+
+- Add XPU to `--torch-backend` ([#14172](https://github.com/astral-sh/uv/pull/14172))
+- Add ROCm backends to `--torch-backend` ([#14120](https://github.com/astral-sh/uv/pull/14120))
+- Add `[tool.uv.dependency-groups].mygroup.requires-python` ([#13735](https://github.com/astral-sh/uv/pull/13735))
+- Add auto-detection for AMD GPUs ([#14176](https://github.com/astral-sh/uv/pull/14176))
+- Show retries for HTTP status code errors ([#13897](https://github.com/astral-sh/uv/pull/13897))
+- Support transparent Python patch version upgrades ([#13954](https://github.com/astral-sh/uv/pull/13954))
+- Warn on empty index directory ([#13940](https://github.com/astral-sh/uv/pull/13940))
+- Publish to DockerHub ([#14088](https://github.com/astral-sh/uv/pull/14088))
+
+### Performance
+
+- Make cold resolves about 10% faster ([#14035](https://github.com/astral-sh/uv/pull/14035))
+
+### Preview features
 
+- Remove preview label from `--torch-backend` ([#14119](https://github.com/astral-sh/uv/pull/14119))
```

Comment:
I'd move this to the top of the enhancements as "Stabilize `--torch-backend`" so it's clear why the rest of the torch-backend items isn't in the preview list.